kubernetes/kubernetes.libsonnet

```diff
@@ -36,6 +36,34 @@ local objectMetadata(object, config) =
     } + config.podAnnotations
   );
 
+// Function to create volumes in deployment
+local volumeService(volumeSpec) = [
+  local volume = k.core.v1.volume;
+  local name = volume.withName(vol.name);
+  volume.withName(vol.name)
+  + (if std.objectHas(vol, 'awsElasticBlockStore') then volume.awsElasticBlockStore.withFsType(vol.awsElasticBlockStore.fsType) else {})
```

Comment:
we don't really use `awsElasticBlockStore` so not really sure if we want to support them. So far we've either had a need for PVC's, secrets or configmaps